src/core/injectHandleError.js

```diff
@@ -10,21 +10,25 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { toError } from "../utils";
+import { toError, updateErrorMessage } from "../utils";
 import { DECLINED_CONSENT_ERROR_CODE } from "./consent/createConsentStateMachine";
 
 export default ({ errorPrefix, logger }) => (error, operation) => {
+  const err = toError(error);
```

Comment:
Moving this line wasn't necessary, but I thought it was safer since it ensures that's we're dealing with an error object before we attempt to access the `code` property below.